### PR TITLE
solution of q 3 and 5

### PR DESCRIPTION
**Pull request by sohamsg8**
I am opening a PR to merge the solutions of qs 3 and 5

### CODE REVIEW

File: code/Assignment_2_q3.Rmd [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+$$
+f(x)=p*Gamma(x|\alpha_1,\sigma_1)+(1-p)Gamma(x|\alpha_2,\sigma_2),~~0<p<1
+$$
+
+(iii) **Model 3**:
+$$
+f(x)=p*logNormal(x|\mu_1,\sigma_1^2)+(1-p)logNormal(x|\mu_1,\sigma_1^2),~~0<p<1
+$$
+```{r}
+library(tidyverse)
```
Comment (ujandasgupta) [inline code/Assignment_2_q3.Rmd]: The libraries tidyverse and scales is unnecessary. Can you remove it?

**Review by ujandasgupta — CHANGES_REQUESTED** [review]
I have run the following code and

File: code/Assignment_2_q3.Rmd (hunk 1 above)
Comment (sohamsg8) [reply code/Assignment_2_q3.Rmd]: sure

**Review by ujandasgupta — APPROVED** [review]
Yes I have verified the changes.

**Review by AbhishekDS1729 — APPROVED** [review]
All seems okay. I approve the changes.

**Pull request merged**